### DISCUSSION TITLE
We are an independent branch campus with a different domain, so my registration is getting stuck.

### DISCUSSION THREAD

**Discussion by starkjeffrey** [community/community#133798] [GitHub Education]
### Hi everyone,

Hi. My school is listed under an associated campus. Our school is pucsr.edu.kh, Pannasastra University of Cambodia, Siem Reap Campus (about 400 km away from the Phnom Penh branch you have registered.). Furthermore, they have misspelled this school name. It is "Pannasastra". What can I do to register my own school, which is an independent entity with separate management. Thanks you. (My email is stark.jeffrey@pucsr.edu.kh and I am Assistant Director of this campus.)

**Comment by SohanKafle** [comment]
Hi [starkjeffrey](https://github.com/starkjeffrey),

To address your issue with registering your independent campus, here are a few steps you can take:

**1. Contact GitHub Education Support:** Explain that Pannasastra University of Cambodia, Siem Reap Campus, is an independent entity with a different domain (pucsr.edu.kh) and separate management. Mention the misspelling of the school name as well. You can reach them [here](https://support.github.com/contact/education).

**2. Provide Detailed Information:** When contacting support, include the correct name, domain, and any relevant documentation that demonstrates your campus's independence from the Phnom Penh branch.

**3. Request an Update:** Ask them to update their records to reflect the correct information and to list your campus separately.

Hopefully, this will help resolve the registration issue.

**Reply by samus-aran** [reply]
This is exactly what you do, well done @SohanKafle 👍
